### PR TITLE
Add change log for 10.2.0-beta.1

### PR DESCRIPTION
**Pull request by Chaoba**
PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).
Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

## Pull request checklist:
 - [ ] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [ ] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog></changelog>`.

### CODE REVIEW

**Review by kiryldz — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,40 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.2.0-beta.1 November 18, 2021
+
+## Features ✨ and improvements 🏁
+* Remove `MapView` argument when constructing `AnnotationManager`. ([#766](https://github.com/mapbox/mapbox-maps-android/pull/766))
```
Comment (pengdev) [inline CHANGELOG.md:8]: shall we mention the previous API with mapView as argument is deprecated?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,40 @@
...
+* Implement cluster API on top of `MapboxMap.queryFeatureExtensions`. ([#773](https://github.com/mapbox/mapbox-maps-android/pull/773))
+* Introduce view annotation support which allows adding Android views on top of the `MapView` anchored to geo-point. ([#834](https://github.com/mapbox/mapbox-maps-android/pull/834))
+* Add heatmap and circle layer support to globe view. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Improve performance for querying large number of features. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Cache layer layout key inside layer, so that it is not re-evaluated at every parse of the every tile. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Model layer now supports loading of external `gltf` files. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
```
Comment (pengdev) [inline CHANGELOG.md:14]: we currently don't expose model layer, so suggest to remove this entry.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,40 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.2.0-beta.1 November 18, 2021
+
+## Features ✨ and improvements 🏁
+* Remove `MapView` argument when constructing `AnnotationManager`. ([#766](https://github.com/mapbox/mapbox-maps-android/pull/766))
+* Implement cluster API on top of `MapboxMap.queryFeatureExtensions`. ([#773](https://github.com/mapbox/mapbox-maps-android/pull/773))
```
Comment (tobrun) [inline CHANGELOG.md:9]: Could use some wording what benefits this brings: eg. "this makes using GeoJson Cluster APIs more easy to use as .."

File: CHANGELOG.md (hunk 2 above)
Comment (kiryldz) [reply CHANGELOG.md:14]: 👍

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,40 @@
...
+* Introduce view annotation support which allows adding Android views on top of the `MapView` anchored to geo-point. ([#834](https://github.com/mapbox/mapbox-maps-android/pull/834))
+* Add heatmap and circle layer support to globe view. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Improve performance for querying large number of features. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
```
Comment (tobrun) [inline CHANGELOG.md:12]: Could use some additional wording on how we were able to do that

File: CHANGELOG.md (hunk 1 above)
Comment (kiryldz) [reply CHANGELOG.md:8]: was thinking about it, but thought about adding this level of details to release notes, WDYT?

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,40 @@
...
+* Cache layer layout key inside layer, so that it is not re-evaluated at every parse of the every tile. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Model layer now supports loading of external `gltf` files. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Add cancelable Query Rendered Features API to `MapboxMap`. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Core renderer prints its version on initialization. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Introduce experimental `MapboxMap.setMemoryBudget` method for setting memory budget for the map and runtime "resource-budget" property for data sources. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Improve performance by avoiding re-layout of invisible fading tiles. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Reduce memory consumption of a model layer by avoiding duplicate texture uploads. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
```
Comment (pengdev) [inline CHANGELOG.md:19]: same as above, we don't expose model layer publicly currently.

Comment (kiryldz) [reply CHANGELOG.md:19]: 👍

**Review by tobrun — APPROVED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,40 @@
...
+* Fix scale bar ratio setting not applied correctly. ([#827](https://github.com/mapbox/mapbox-maps-android/pull/827))
+* Fix scale bar text missing for Android API 23. ([#839](https://github.com/mapbox/mapbox-maps-android/pull/839))
+* Fix scale bar text being overlapped and clipped. ([#856](https://github.com/mapbox/mapbox-maps-android/pull/856))
+* Fix puck jump to nullisland when location plugin settings are changed. ([#846](https://github.com/mapbox/mapbox-maps-android/pull/846))
+* Fix scale listener events not being called for quick zoom doubleTap and doubleTouch gestures. ([#858](https://github.com/mapbox/mapbox-maps-android/pull/858))
+* Release all unused resources when `MapboxMap.reduceMemoryUse` is invoked. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Fix crash for the case when an empty fill extrusion bucket is tried to be rendered. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Fix transparency issues with value < 0.5 for models rendered by the model layer. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
```
Comment (pengdev) [inline CHANGELOG.md:30]: As model layer isn't exposed, we could describe the impact for 3D location puck.

File: CHANGELOG.md (hunk 4 above)
Comment (kiryldz) [reply CHANGELOG.md:12]: I have no idea how we were able to do that, it's about actually speeding up QRF function dramatically.

**Review by pengdev — APPROVED** [review]
:shipit:

**Review by alexander-kulikovskii — APPROVED** [review]

**Pull request merged**